cirq/linalg/transformations.py

```diff
@@ -149,6 +149,45 @@ def targeted_left_multiply(left_matrix: np.ndarray,
                      **({'out': out} if out is not None else {}))
 
 
+def targeted_conjugate(
+    tensor: np.ndarray,
+    target: np.ndarray,
+    indices: Sequence[int],
+    conj_indices: Sequence[int],
+    out: Optional[np.ndarray] = None) -> np.ndarray:
+    """Conjugates the given tensor about the target tensor.
```

Comment:
I think we need the expression "ABA*" somewhere in this summary to indicate we mean a different type of conjugation than negating the imaginary component.

Reply:
expanded documentation